packet facilities get fails when a facility is not assigned to metro (staff API key)

### What happened?

When using a staff API key, `packet facilities get` fails. Likely cause is facilities like `sg4` that are only staff viewable that have `"metro": null` as they don't belong to a metro, but facilities requires a metro to build the table.

### How can we reproduce it?

```
% packet facilities get

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x146cbc3]
goroutine 1 [running]:
github.com/packethost/packet-cli/cmd.glob..func32(0x1990020, 0x19c44f0, 0x0, 0x0, 0x0, 0x0)
	/home/runner/work/packet-cli/packet-cli/cmd/retrieve_facilities.go:48 +0x223
github.com/spf13/cobra.(*Command).execute(0x1990020, 0x19c44f0, 0x0, 0x0, 0x1990020, 0x19c44f0)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:842 +0x453
github.com/spf13/cobra.(*Command).ExecuteC(0x198f840, 0x0, 0x14b69c0, 0xc00008e058)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:950 +0x349
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:887
main.main()
	/home/runner/work/packet-cli/packet-cli/main.go:31 +0x33
```

CLI version (`packet --version`):  packet version 0.4.0

packet-darwin-amd64 binary on macOS 11.3.1

--- a/cmd/retrieve_facilities.go
+++ b/cmd/retrieve_facilities.go
@@ -45,7 +45,11 @@ packet facilities get
 		data := make([][]string, len(facilities))
 
 		for i, facility := range facilities {
-			data[i] = []string{facility.Name, facility.Code, facility.Metro.Code, strings.Join(facility.Features, ",")}
+			var metro string
+			if facility.Metro != nil {
+				metro = facility.Metro.Code
+			}
+			data[i] = []string{facility.Name, facility.Code, metro, strings.Join(facility.Features, ",")}
 		}
 		header := []string{"Name", "Code", "Metro", "Features"}
 

--- a/go.mod
+++ b/go.mod
@@ -14,7 +14,7 @@ require (
 	github.com/mattn/go-isatty v0.0.3 // indirect
 	github.com/mattn/go-runewidth v0.0.2 // indirect
 	github.com/olekukonko/tablewriter v0.0.0-20180506121414-d4647c9c7a84
-	github.com/packethost/packngo v0.10.0
+	github.com/packethost/packngo v0.15.0
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/cobra v1.0.0
 	github.com/spf13/viper v1.4.0

